jicoco/src/main/kotlin/org/jitsi/rest/JettyHelpers.kt

```diff
@@ -17,32 +17,32 @@
 
 package org.jitsi.rest
 
-import org.eclipse.jetty.server.HttpConfiguration
-import org.eclipse.jetty.server.HttpConnectionFactory
-import org.eclipse.jetty.server.SecureRequestCustomizer
-import org.eclipse.jetty.server.Server
-import org.eclipse.jetty.server.ServerConnector
-import org.eclipse.jetty.server.SslConnectionFactory
+import org.eclipse.jetty.server.*
 import org.eclipse.jetty.servlet.ServletContextHandler
 import org.eclipse.jetty.servlet.ServletHolder
 import org.eclipse.jetty.servlets.CrossOriginFilter
 import org.eclipse.jetty.util.ssl.SslContextFactory
 import org.jitsi.utils.getJavaVersion
 import java.nio.file.Paths
-import java.util.EnumSet
+import java.util.*
 import javax.servlet.DispatcherType
 
 /**
  * Create a non-secure Jetty server instance listening on the given [port] and [host] address.
+ * [sendServerVersion] controls whether Jetty should send its server version in the error responses or not.
  */
 fun createJettyServer(
     port: Int,
-    host: String? = null
+    host: String? = null,
+    sendServerVersion: Boolean = true
```

Comment:
Hm, I'm not sure we want a default value here, as we've now got two places where the default lives.  Thoughts @bgrozev?  I see we've got some default values for various params in here already.

Reply:
I think it's fine either way since its just a function param.